### PR TITLE
Update Wolvesville

### PR DESCRIPTION
**Pull request by down-bad**
Revamped settings, added more statuses, fixed showing 0/0 players alive, localized presence description (official translations from the game translators), and other smaller changes.

### CODE REVIEW

File: websites/W/Wolvesville/presence.ts [hunk 1]
```diff
@@ -124,14 +137,14 @@ presence.on("UpdateData", async () => {
     if (menu) {
       data.details = "At the main menu";
 
-      if (showUsername) {
+      if (privacyMode === false) {
```
Comment (EncryptedDev) [inline websites/W/Wolvesville/presence.ts:140]:
```suggestion
      if (!privacyMode) {
```

File: websites/W/Wolvesville/presence.ts [hunk 2]
```diff
@@ -124,14 +137,14 @@ presence.on("UpdateData", async () => {
...
         const username = document.querySelectorAll(
           "div.css-901oao.r-jwli3a.r-ubezar.r-5oul0u"
         );
         data.state = username[0]?.textContent;
       }
 
-      if (showMenuState) {
+      if (privacyMode === false) {
```
Comment (EncryptedDev) [inline websites/W/Wolvesville/presence.ts:147]: Ditto

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/W/Wolvesville/presence.ts (hunk 1 above)
Comment (down-bad) [reply websites/W/Wolvesville/presence.ts:140]: That's how I had them in the beginning but it didn't work properly (spent like an hour trying to figure out why). I'll try testing it again though.

Comment (down-bad) [reply websites/W/Wolvesville/presence.ts:140]: Nvm works now, idk what was wrong with it when I tested it some weeks ago

**Comment by EncryptedDev** [comment]
Resolve ESlint issues by running `eslint .` in the service folder

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**